packages/dmn-js-decision-table/src/features/decision-table-head/components/DecisionTableHead.js

```diff
@@ -143,7 +143,7 @@ function DefaultInputHeaderCell(props, context) {
 
       {
         label ? (
-          <div className="input-label" title={ translate('Input Label') }>
```

Comment:
Shall we keep the prefix `Input Label` here? The intention would be to still tell users what this is, but also provide full details: `Input Label - SOME TEXT THAT I ENTERED AS A USER` for example.

Reply:
Makes sense, I just noticed we do it for the decision Name already
https://github.com/bpmn-io/dmn-js/blob/3081fd9284520120043697de51bfa5fdd8fb57fc/packages/dmn-js-decision-table/src/features/decision-table-properties/components/DecisionTablePropertiesComponent.js#L31

Reply:
Adjusted with [6466c76](https://github.com/bpmn-io/dmn-js/pull/736/commits/6466c7682767f8d4b595f7d3877ec15b5b156d33)